packages/flutter/lib/src/widgets/icon_theme_data.dart

```diff
@@ -87,6 +90,9 @@ class IconThemeData with Diagnosticable {
   /// The default size for icons.
   final double? size;
 
+  /// The default splash radius for icons.
```

Comment:
This should mention IconButton since it only applies to that widget. Something like:

```
/// The default splash radius for [IconButton]s
```

Reply:
Done

Reply:
Oops, looks like you modified the comment for the `size` property instead of the splashRadius